<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>54</y>
      <width>211</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QTextEdit" name="EnterIP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>–í–≤–µ–¥–∏—Ç–µ IP –∞–¥—Ä–µ—Å ...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AddIP">
     <property name="geometry">
      <rect>
       <x>168</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>‚ûï</string>
     </property>
    </widget>
    <widget class="QListWidget" name="ServerList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>499</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="SoundOnOff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>6</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>üéß</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MicOnOff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>26</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>üéôÔ∏è</string>
     </property>
    </widget>
    <widget class="QSlider" name="ThresholdVolume">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>47</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="MicVolume">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>28</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="SoundVolume">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>8</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QTextEdit" name="EnterName">
     <property name="geometry">
      <rect>
       <x>73</x>
       <y>5</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>11</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>–í–∞—à–µ –∏–º—è:</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>–°–ø–∏—Å–æ–∫ —É—á–∞—Å—Ç–Ω–∏–∫–æ–≤:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="UserList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>419</y>
      <width>571</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QTextEdit" name="EnterMessage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>113</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SendMessage">
     <property name="geometry">
      <rect>
       <x>484</x>
       <y>115</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>–û—Ç–ø—Ä–∞–≤–∏—Ç—å</string>
     </property>
    </widget>
    <widget class="QListWidget" name="MessageList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>554</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>571</width>
      <height>401</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
